src/extremeweatherbench/config.py

```diff
@@ -2,7 +2,7 @@
 
 import dataclasses
 from pathlib import Path
-from typing import Callable, List, Optional
+from typing import Callable, List, Optional, Union
```

Comment:
In Python >= 3.10, you can just use the "|" operator. So `Union[str, Path]` is equivalent to `str | Path` (or most commonly, you would see `str | None`).